src/components/CustomInputs/DictInput.vue

```diff
@@ -224,6 +248,9 @@ export default {
                   ? dict[i].value.toString()
                   : valueLabel
               "
+              :readonly="
```

Comment:
Is there a reason this is readonly instead of disabled?  My preference would be to use disabled here as it's clearer that it's deliberately disabled.  Read-only feels odd as I can focus on it but not change the value.

Reply:
This was primarily to match styling of the key field, but I'm not sure the distinction between readonly and disabled is particularly important when the field is disabled either way